### ISSUE TITLE
Device handlers are completely ignored

### ISSUE THREAD

**Issue by nazar-pc** [IgnorantGuru/spacefm#714]
I've built GTK3 version from `next` branch just to doublecheck (commit 6b37a55) and behavior is the same:
* there is `/dev/sdc1`
* default device handler is set to `udisksctl mount -b %v`

### Expected result
Partition is mounted to `/media/nazar-pc/nazar-pc` because of user and partition names (and this indeed happens when running `udisksctl mount -b /dev/sdc1`).

### Actual result
Partition is mounted to `/media/sdc1` regardless of device handlers settings (even disabling default handler have no effect).

**Comment by IgnorantGuru** [comment]
Start the initial instance of spacefm in a terminal so you can see the output.  This should give you more information on what device handler is being used and why.  You can provide that output here if you'd like help with it.  There is nothing mysterious about how SpaceFM mounts devices - you can see the exact commands being run in the terminal output.  Selection of device handlers is described in the manual.

**Comment by nazar-pc** [comment]
```
TASK_COMMAND(0x5564423883f0)=/usr/bin/pmount /dev/sdc1
SPAWN=/bin/bash  /tmp/spacefm-nazar-pc-7a1166cb.tmp/1922e725-tmp.sh  run
    pid = 30011
mount changed: /dev/sdc1
child finished  pid=30011 exit_status=0
```

It does ignore my settings:(

**Comment by IgnorantGuru** [comment]
If the output shows nothing about a device handler being selected, iirc that means no device handler was available.  In this case, SpaceFM will use udevil, pmount, or udisks, depending on what is installed.  In this case, it's falling back to pmount.

I suggest re-enabling the default device handler, add your preferred mount command there, and then see how the output changes.

There haven't been any significant changes to device handlers in a couple years, and no reports of malfunctions, so it's likely just a configuration issue.

**Comment by nazar-pc** [comment]
Disabled, re-enabled, no difference.
```
nazar-pc@nazar-pc ~> cat ~/.config/spacefm/session 
...
hand_fs_+def-s=*
hand_fs_+def-x=
hand_fs_+def-label=Default
hand_fs_+def-b=1
...
nazar-pc@nazar-pc ~> cat ~/.config/spacefm/scripts/hand_fs_+def/hand-device-mount.sh 
#!/bin/bash

udisksctl mount -b %v
```

**Comment by IgnorantGuru** [comment]
Yes there does seem to be some kind of a problem (testing alpha).  It seems the 'udevil iso unmount' handler is causing it, so if you disable that or move it below Default, then the Default handler should be used.  I'll take a look at what's going on there shortly.
